auraed/src/runtime/cells/cell.rs

```diff
@@ -129,3 +189,48 @@ fn hierarchy() -> Box<dyn Hierarchy> {
     // hierarchies::V2
     Box::new(hierarchies::V2::new())
 }
+
+#[derive(Error, Debug)]
+pub(crate) enum CellError {
+    #[error("cell '{cell_name}' already exists'")]
+    Exists { cell_name: CellName },
```

Comment:
nit: "CellExists" to parallel ExecutableExists ?

Reply:
I'm torn. Will go with your suggestion.